gulp-core-build/src/tasks/ValidateShrinkwrapTask.ts

```diff
@@ -6,11 +6,17 @@ import * as semver from 'semver';
 type ShrinkwrapDep = { [name: string]: { version: string } };
 type PackageDep = { [name: string]: string };
 
+/**
+ * Partial representation of the contents of a `package.json` file
```

Comment:
Partial, meaning that the this is not an interface to represent the entire package.json file.